frontend/src/components/workspaces/Templates/TemplatesTable/TemplatesTable.tsx

```diff
@@ -96,35 +63,34 @@ const TemplatesTable: FC<ITemplatesTableProps> = ({ ...props }) => {
       className="w-full flex-grow flex-wrap content-between py-0 overflow-auto scrollbar"
       style={{ height: 'calc(100vh - 380px)', minHeight: '171px' }} //465px if we want to add a Box footer
     >
-      <Col span={24} className="flex-auto">
-        <Table
-          size={'small'}
-          showHeader={false}
-          rowKey={record => record.id}
-          columns={columns}
-          dataSource={templates}
-          pagination={false}
-          onExpand={handleAccordion}
-          expandable={{
-            rowExpandable: record => !!record.instances.length,
-            expandedRowKeys: expandedId,
-            expandIcon: ({ expanded, onExpand, record }) =>
-              record.instances.length ? (
-                expanded ? (
-                  <DownOutlined onClick={e => onExpand(record, e)} />
-                ) : (
-                  <RightOutlined onClick={e => onExpand(record, e)} />
-                )
+      {/*removed col*/}
+      <Table
+        size={'small'}
+        showHeader={false}
+        rowKey={record => record.id}
+        columns={columns}
+        dataSource={templates}
+        pagination={false}
+        onExpand={handleAccordion}
+        expandable={{
+          rowExpandable: record => !!record.instances.length,
+          expandedRowKeys: expandedId,
+          expandIcon: ({ expanded, onExpand, record }) =>
+            record.instances.length ? (
+              expanded ? (
+                <DownOutlined onClick={e => onExpand(record, e)} />
               ) : (
-                false
-              ),
-            /**
-             * Here we render the expandable content, for example with a nested Table
-             */
-            expandedRowRender: template => 'Running Instances',
-          }}
-        />
-      </Col>
+                <RightOutlined onClick={e => onExpand(record, e)} />
+              )
+            ) : (
+              false
+            ),
```

Comment:
Isn't the `expr && elm` "pattern" working here?

Reply:
it was an overview, i'll test and correct it, thanks

Reply:
UPDATE: i test you change suggestion but it not work because the new expression is rendered as `0` was the result is `false`. I think i have to leave everything unchanged

Reply:
That's what I feared